services/src/messaging/src/main/java/org/openkilda/messaging/command/flow/BatchFlowInstallForSwitchManagerRequest.java

```diff
@@ -26,15 +26,15 @@
 
 @Value
 @Builder
-public class BatchInstallForSwitchManagerRequest extends CommandData {
+public class BatchFlowInstallForSwitchManagerRequest extends CommandData {
```

Comment:
I would suggest to name `BatchInstallFlowForSwitchManagerRequest` here to match the overall style.

Reply:
Yep, rename reverted